### ISSUE TITLE
Reconciliation over OpenSearch

### ISSUE THREAD

**Issue by Abbe98** [OpenRefine/OpenRefine#5472]
By supporting OpenSearch endpoints as reconciliation services one would greatly existing the number of reconciliation services available. OpenSearch would also be a low barrier alternative to Reconciliation Service API for service development.

https://en.wikipedia.org/wiki/OpenSearch

**Comment by wetneb** [comment]
Absolutely! That is something we have had on our radar in the [reconciliation community group](https://www.w3.org/community/reconciliation/):
https://reconciliation-api.github.io/census/other-protocols/#opensearch

Independently of the spec work happening over there it could make sense to add support for OpenSearch services for reconciliation in OpenRefine.

**Comment by thadguidry** [comment]
But there's no scoring, only results with OpenSearch?  How would that work?  Scoring is implemented client side then, if the results provide enough disambiguating properties for each candidate?  How a match is determined, algorithmically or human, is pretty important, and scoring has to be done by one or the other or both.

**Comment by Abbe98** [comment]
I usually just interfere scoring from the search result order or set all results to the same score.
